src/library_addfunction.js

```diff
@@ -238,6 +238,7 @@ addToLibrary({
   $removeFunction__deps: ['$functionsInTableMap', '$freeTableIndexes', '$getWasmTableEntry'],
   $removeFunction: (index) => {
     functionsInTableMap.delete(getWasmTableEntry(index));
+    setWasmTableEntry(index, null);
```

Comment:
Please add this to line `238` above as this is a new dependency.

Reply:
makes sense, updated